web/src/validators/social-media-urls.js

```diff
@@ -0,0 +1,18 @@
+import { helpers } from 'vuelidate/lib/validators';
+
+const linkedinUrl = (value) => {
+  const regex = /^https?:\/\/((www|\w\w)\.)?linkedin.com\/((in\/[^/]+\/?)|(pub\/[^/]+\/((\w|\d)+\/?){3}))$/;
+  return !helpers.req(value) || regex.test(value);
+};
+
+const facebookUrl = (value) => {
+  const regex = /^http(s*):\/\/(www.)*facebook.com\/[a-zA-Z0-9.]+$/;
+  return !helpers.req(value) || regex.test(value);
+};
+
+const twitterUrl = (value) => {
+  const regex = /^http(s*):\/\/(www.)?twitter.com\/([a-zA-Z0-9_]+)+$/;
```

Comment:
Could support a trailing slash here as well, as twitter still resolves to the profile page

Reply:
@ann-kilzer twitter url should also support trailing / now